packages/data-stores/src/domain-suggestions/utils.ts

```diff
@@ -30,3 +34,40 @@ export function getFormattedPrice( price: number, currencyCode: string ): string
 		stripZeros: true,
 	} ) as string;
 }
+
+/**
+ * Normalize domain query
+ *
+ * It's important to have a consistent, reproduceable representation of a domains query so that the result can be
+ * stored and retrieved.
+ *
+ * @see client/state/domains/suggestions/utils.js
+ * @see client/components/data/query-domains-suggestions/index.jsx
+ *
+ * @param search       Domain search string
+ * @param queryOptions Optional paramaters for the query
+ * @returns Normalized query object
+ */
+export function normalizeDomainSuggestionQuery(
+	search: string,
+	queryOptions: DomainSuggestionSelectorOptions
+): DomainSuggestionQuery {
+	return {
+		// Defaults
+		include_wordpressdotcom: queryOptions.only_wordpressdotcom || false,
+		include_dotblogsubdomain: false,
+		only_wordpressdotcom: false,
+		quantity: 5,
+		// This is commented out due to cyclic package dependency compilation error.
+		// Since domain picker passes in the vendor prop it is safe to hardcode this for now.
+		// We can re-enable this in a separate PR.
+		// vendor: getDomainSuggestionsVendor(),
+		vendor: 'variation2_front',
```

Comment:
I'm starting to think there's no way to have `data-stores` depend on `domain-picker` and `domain-picker` depend on `data-stores`, I think it is fine to hardcode `variation2_front` here as it is the default value and this value i overridden by domain picker itself. Gonna remove the comments in a bit.

Reply:
It sounds reasonable. I don't think it hurts to have a default value.

An alternative would be to create another package which stores all of these values and makes them available to calypso / other packages

Reply:
Now hardcoded https://github.com/Automattic/wp-calypso/pull/50761/commits/fa14b183e6d44d1fee7f919cb27348950382fbf0.